README.md

```diff
@@ -75,6 +75,28 @@ The RPM carries out a number of steps to run the SSM in a specific way.
 6. It creates the pidfile directory /var/run/apel/
 7. It installs a service script in /etc/init.d/
 
+## Installing the DEB
+
+### Installation
+Install APEL SSM.
+* `dpkg -i apel-ssm_<version>_all.deb`
+
+Install any missing system packages needed for the SSM.
+* `apt-get -f install`
+
+Install any missing python requirements that don't have system packages.
+* `pip install "stomp.py>=3.1.1" python-daemon python-ldap dirq`
```

Comment:
None of these have Ubuntu packages?

Reply:
stomp.py, no - there is a confusingly named python-stompy but its a complete different library
dirq - seemingly not

there does look like there are suitable packages for python-ldap and python-daemon, so I'll use them.

Reply:
Mostly I just wanted to understand the differences. Can discuss later.

Reply:
Ah I remember now, the python `python-ldap` requires the system `python-ldap` headers in order to install. That's why it's listed here and in the build script as a dependency. 

And python-daemon is an optional requirement, so should stay out of the package dependencies as per #70.

I'll reword this line to be clearer

Reply:
Infact, system `python-ldap` seems to be sufficient!